templates/kotlin/docs/java/example.md.twig

```diff
@@ -36,11 +36,11 @@ public void main() {
 
 {% for parameter in method.parameters.all %}
 {% if parameter.required %}
-        {{parameter.name}} = {{ parameter | paramExample }}{% if not loop.last %},{% endif %}
+        {{ parameter | paramExample }}{% if not loop.last %},{% endif %}
```

Comment:
@abnegate we wanted the named params?

Reply:
@lohanidamodar Unfortunately Java does not support named parameters